custom_dc/quickstart.md

```diff
@@ -119,19 +116,19 @@ Note: If you are storing your source code in a public/open-source version contro
 docker run -it \
 -p 8080:8080 \
 -e DEBUG=true \
---env-file $PWD/custom_dc/sqlite_env.list \
--v $PWD/custom_dc/sample:/userdata \
+--env-file $PWD/custom_dc/env.list \
+-v $PWD/custom_dc/sample:/$PWD/custom_dc/sample  \
```

Comment:
Remove the slash after the colon.

Can you go through all `docker run` commands in the doc to make sure both sides of the colon are the same exact paths?

Reply:
Done.